op-node/service.go

```diff
@@ -344,15 +345,15 @@ func NewL1ChainConfigFromCLI(log log.Logger, ctx *cli.Context) (*params.ChainCon
 	return jsonutil.LoadJSONFieldStrict[params.ChainConfig](l1ChainConfigPath, "config")
 }
 
-func NewDependencySetFromCLI(ctx *cli.Context) (depset.DependencySet, error) {
-	if !ctx.IsSet(flags.InteropDependencySet.Name) {
+func NewDependencySetFromCLI(cli cliiface.Context, ctx context.Context) (depset.DependencySet, error) {
```

Comment:
```suggestion
func NewDependencySetFromCLI(ctx context.Context, cli cliiface.Context) (depset.DependencySet, error) {
```

^ Should we pass `ctx context.Context` first to be more idiomatic?

Reply:
no, I'd like to have all access to the flags remain interfaced. Also this "New Dependency Set From CLI" is old interop code which may become deprecated so I'm not too worried about it one way or another.